src/UI/Components/ServiceInstanceForm/Components/FieldInput.tsx

```diff
@@ -79,6 +95,26 @@ export const FieldInput: React.FC<Props> = ({
     [getUpdate, path, field.name],
   );
 
+  useEffect(() => {
+    if (!isLoading && !error) {
+      // if the data is of type array, we can use it directly
+      if (Array.isArray(data)) {
+        setSuggestionsList(data);
+      } else if (
+        data &&
+        data.metadata &&
+        data.metadata.values &&
+        Array.isArray(data.metadata.values) &&
+        // TODO: remove this when the API returns a fixed format.
```

Comment:
Should we assign to to its own ticket?

Reply:
I'll make a ticket, yes

Reply:
https://github.com/inmanta/web-console/issues/5582